README.md

```diff
@@ -13,7 +14,12 @@ run mdformat on your own. CI will fail if markdown files are not correctly
 formatted!
```

Comment:
It says 'run mdformat' here, shouldn't it be run Prettier?

Reply:
Good catch, updated

Reply:
Can we add a link for how to enable experimental features (if someone wants to)?

Reply:
Sure, I think that is kind of marginal (the docs here shouldn't be responsible for teaching someone how to use nix, they just offer a method of setting up the project for nix users) but I am happy to provide a link.